package.json

```diff
@@ -1,52 +1,53 @@
 {
-  "name": "express-authorshaven",
-  "version": "1.0.0",
-  "description": "A Social platform for the creative at heart",
-  "main": "index.js",
-  "scripts": {
-    "start": "babel-node src/index.js",
-    "test": "NODE_ENV=test nyc --reporter=html --reporter=text --reporter=lcov ./node_modules/.bin/mocha  ./test/* --require @babel/register --exit",
-    "dev": "nodemon --exec babel-node src/index.js",
-    "coverage": "nyc report --reporter=text-lcov | coveralls"
-  },
-  "devDependencies": {
-    "chai": "^4.2.0",
-    "chai-http": "^4.3.0",
-    "coveralls": "^3.0.3",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.17.3",
-    "mocha": "^6.1.4",
-    "nodemon": "^1.19.1",
-    "nyc": "^14.1.1"
-  },
-  "dependencies": {
-    "babel-plugin-istanbul": "^5.1.2",
-    "@babel/cli": "^7.4.4",
-    "@babel/core": "^7.4.5",
-    "@babel/node": "^7.4.5",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/register": "^7.4.4",
-    "body-parser": "^1.18.3",
-    "chai-http": "^4.3.0",
-    "cors": "^2.8.5",
-    "dotenv": "^8.0.0",
-    "expect": "^24.8.0",
-    "express": "^4.16.4",
-    "morgan": "^1.9.1",
-    "pg": "^7.11.0",
-    "sequelize": "^5.8.7",
-    "swagger-node-express": "^2.1.3",
-    "swagger-ui-express": "^4.0.6"
-  },
-  "nyc": {
-    "per-file": true,
-    "check-coverage": true,
-    "branches": 50,
-    "lines": 80,
-    "functions": 80,
-    "statements": 80
-  },
-  "author": "Andela Simulations Programme",
-  "license": "MIT"
+	"name": "express-authorshaven",
+	"version": "1.0.0",
+	"description": "A Social platform for the creative at heart",
+	"main": "index.js",
+	"scripts": {
+		"start": "babel-node src/index.js",
+		"test": "NODE_ENV=test nyc --reporter=html --reporter=text --reporter=lcov ./node_modules/.bin/mocha  ./tests/* --require @babel/register --exit",
+		"dev": "nodemon --exec babel-node src/index.js",
+		"coverage": "nyc report --reporter=text-lcov | coveralls"
+	},
+	"devDependencies": {
+		"chai": "^4.2.0",
+		"chai-http": "^4.3.0",
+		"coveralls": "^3.0.3",
+		"eslint": "^5.16.0",
+		"eslint-config-airbnb-base": "^13.1.0",
+		"eslint-plugin-import": "^2.17.3",
+		"mocha": "^6.1.4",
+		"nodemon": "^1.19.1",
+		"nyc": "^14.1.1"
+	},
+	"dependencies": {
+		"@babel/cli": "^7.4.4",
+		"@babel/core": "^7.4.5",
+		"@babel/node": "^7.4.5",
+		"@babel/preset-env": "^7.4.5",
+		"@babel/register": "^7.4.4",
+		"babel-plugin-istanbul": "^5.1.2",
+		"body-parser": "^1.18.3",
+		"chai-http": "^4.3.0",
+		"cors": "^2.8.5",
+		"dotenv": "^8.0.0",
+		"expect": "^24.8.0",
+		"express": "^4.16.4",
+		"joi": "^14.3.1",
+		"morgan": "^1.9.1",
+		"pg": "^7.11.0",
+		"sequelize": "^5.8.7",
+		"swagger-node-express": "^2.1.3",
+		"swagger-ui-express": "^4.0.6"
+	},
+	"nyc": {
+		"per-file": true,
+		"check-coverage": true,
+		"branches": 50,
```

Comment:
The threshold should be 80 for branches and 100 for functions. May you change these parameters and make sure your tests are passing?